spec/spec_helper.rb

```diff
@@ -2,14 +2,16 @@
 
 require 'factory_bot'
 require 'simplecov'
-Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
+
+Dir[Pathname(__dir__).join('support', '**', '*.rb')].each { |f| require f }
```

Comment:
wassthis

Reply:
Because it's in the `spec_helper`, `Rails` isn't a thing (yet)